config/samples/config_v1alpha1_verifier_sample.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: config.ratify.deislabs.io/v1alpha1
+kind: Verifier
+metadata:
+  name: verifier-sample
+spec:
+  name: sample
+  artifactTypes: application/spdx+json
+  source: noelfdpo.azurecr.io/sample-plugin:v1
```

Comment:
This should be replaced with a known-good "official" source. Maybe all plugins should point to `ghcr.io/ratify/plugins/sbom:v1.0.0-beta.2`, and so on?